Sources/Functions/FunctionsClient.swift

```diff
@@ -130,33 +128,101 @@ public actor FunctionsClient {
   private func rawInvoke(
     functionName: String,
     invokeOptions: FunctionInvokeOptions
-  ) async throws -> (Data, HTTPURLResponse) {
+  ) async throws -> Response {
+    var request = Request(
+      path: functionName,
+      method: .post,
```

Comment:
This line is causing a major bug in the library.  Upon upgrading our client version, we can no longer make GET/PUT/DELETE requests.

Reply:
Hi @jareddr thanks for pointing out the issue, a PR with the fix https://github.com/supabase/supabase-swift/pull/367

I'll get it merged and released ASAP